ros_gz_sim/CMakeLists.txt

```diff
@@ -121,7 +121,10 @@ install(FILES
 
 install(FILES
   "launch/gz_server.launch.py"
+  "launch/gz_spawn_model.launch.py"
   "launch/ros_gz_sim.launch.py"
+  "launch/ros_gz_spawn_model.launch.py"
+
```

Comment:
remove this empty line

Reply:
Removed in [54c68d7](https://github.com/gazebosim/ros_gz/pull/534/commits/54c68d7f3ee5daeba6c9735129fe9c08a0613e42)